concourse/verify-production-urls/task.yml

```diff
@@ -0,0 +1,11 @@
+---
+platform: linux
+
+inputs:
+- name: release
```

Comment:
i think you also need to have `rex-web` as an input just because that is where the script lives that you run down there on line 11